math/power/powvialogarithm_test.go

```diff
@@ -0,0 +1,44 @@
+// powvialogarithm_test.go
+// description: Test for PowUseLog
+// author(s) [red_byte](https://github.com/i-redbyte)
+// see powvialogarithm.go
+
+package power
+
+import "testing"
+
+var tests = []struct {
```

Comment:
Same as previous comment - could you move this to a local declaration/initialisation or factory generation instead? Same point as other test comment applies 😄

Reply:
Done